### PR TITLE
[cli] Update cli flow version to 0.144.0

### PR DESCRIPTION
**Pull request by Brianzchen**
I thought it would make sense that the tool that maintains our type defs  aren't running on a flow version that's just a year old at this point.

- Links to documentation: N/A
- Link to GitHub or NPM: N/A
- Type of contribution: refactor

Other notes:

### CODE REVIEW

File: cli/src/commands/create-stub.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {findFlowRoot} from '../lib/flowProjectUtils.js';
 import {getPackageJsonData, loadPnpResolver} from '../lib/npm/npmProjectUtils';
 import {path} from '../lib/node';
 
-export function setup(yargs: Object) {
+export function setup(yargs: Object): any {
```
Comment (pascalduez) [inline cli/src/commands/create-stub.js:11]: If we could also kill those `Object`, would be nice.

File: cli/src/lib/__mocks__/node.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const fs = {
   // This is a custom function that our tests can use during setup to specify
   // what the files on the "mock" filesystem should look like when any of the
   // `fs` APIs are used.
-  mockFiles: {},
+  mockFiles: ({ ...null }: { [key: string]: any }),
```
Comment (pascalduez) [inline cli/src/lib/__mocks__/node.js:16]: Looks like this breaks the CI, also could you add a comment, that `{...null}` is not straightforward...

File: cli/src/lib/libDefs.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import semver from 'semver';
 import {cloneInto, rebaseRepoMaster} from './git.js';
 import {mkdirp} from './fileUtils.js';
 import {fs, path, os} from './node.js';
-import {versionToString} from './semver.js';
+import {versionToString, type VersionRange, type Version} from './semver.js';
```
Comment (pascalduez) [inline cli/src/lib/libDefs.js:8]: CI break, `VersionRange` unused.

File: cli/src/lib/__mocks__/node.js (hunk 2 above)
Comment (Brianzchen) [reply cli/src/lib/__mocks__/node.js:16]: actually don't need `...null` anymore. Needed it before when it was untyped and was exported an as unsealed object

File: cli/src/commands/create-stub.js (hunk 1 above)
Comment (Brianzchen) [reply cli/src/commands/create-stub.js:11]: I'll look into it

**Review by GAntoine — APPROVED** [review]

**Pull request merged**